index.html

```diff
@@ -1461,7 +1461,7 @@ <h2>Capabilities</h2>
   <td>"<code>acceptInsecureCerts</code>"
   <td>boolean
   <td>Indicates whether untrusted and self-signed TLS certificates
-   are implicitly trusted on <a data-lt=go>navigation</a>
```

Comment:
just noting this was incorrect.
Go was renamed in #1040 so this should be to `navigate to` 
this now goes to `navigate` which is a completely different definition

Reply:
Good catch, @jtnord! It looks like there are actually a bunch of incorrect links for "navigation" leading to https://w3c.github.io/webdriver/#dfn-navigating, I'll file an issue.

Reply:
I think this was wrong before and is now correct. Self-signed certs should be trusted irrespective of whether the navigation was via the `navigate` command or any other method (e.g. synthesizing a click on a link).

Reply:
Whilst this may now be the correct definition, it was not intended to be and this change was an accident.  I should have really commented on [before-unload prompts -line8619](https://github.com/w3c/webdriver/pull/1518/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L8619) and commented on the first change I found instead.  That said I still believe these accidental changes should be reverted, and *then* you can have the discussion about if it was correct before or not and make that change.

Reply:
@jgraham based on our findings here  #1574 got closed. So shall we also resolve this particular conversation too? We should rather continue the discussion about `beforeunload` prompts via #1294.